### PR TITLE
feat: add replay failed transactions

### PR DESCRIPTION
**Pull request by 0xArdy**
closes SUP-78

### CODE REVIEW

File: src/utils/eventLogs/formatWithdrawalLogs.ts [hunk 1]
```diff
@@ -4,94 +4,117 @@ import { AccountLogs, CustomClients } from '~/types';
...
-    from: log.args.from!,
-    to: log.args.to!,
-    amount: log.args.amount!,
-    localToken: log.args.localToken!,
-    remoteToken: log.args.remoteToken!,
-    receipt: statusMap[log.transactionHash].receipt,
-  }));
+  const accountLogs: AccountLogs[] = logs.map((log) => {
+    const { args, msgHash } = msgData[log.transactionHash];
+    const isFailed = failedLogs.some((failedLog) => failedLog.args.msgHash === msgHash);
```
Comment (turtlemoji) [inline src/utils/eventLogs/formatWithdrawalLogs.ts:61]: where do you use `msgHash` and `isFailed`?

File: src/utils/eventLogs/withdrawLogs.ts [hunk 2]
```diff
@@ -22,79 +22,93 @@ export const getWithdrawLogs = async ({
...
-    accountLogs,
-    msgHashes,
-    failedTxs,
-    args,
-  };
+    return {
+      accountLogs,
+    };
+  } catch (error) {
+    console.warn(error);
```
Comment (turtlemoji) [inline src/utils/eventLogs/withdrawLogs.ts:108]: should be a warn or an .error?

**Review by turtlemoji — APPROVED** [review]
pre-approved

File: src/utils/eventLogs/formatWithdrawalLogs.ts (hunk 1 above)
Comment (0xArdy) [reply src/utils/eventLogs/formatWithdrawalLogs.ts:61]: here, to let the user replay the transaction https://github.com/defi-wonderland/superchain-bridge-ui/pull/41/files#diff-ddcd466a3e8c425587f5a4809ae48f12079bfede0c1dfd508eb1a75edeeeecac

File: src/utils/eventLogs/withdrawLogs.ts (hunk 2 above)
Comment (0xArdy) [reply src/utils/eventLogs/withdrawLogs.ts:108]: warn

**Pull request merged**